api/envoy/extensions/access_loggers/fluentd/v3/fluentd.proto

```diff
@@ -22,8 +22,42 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // the Fluentd Forward Protocol as described in: `Fluentd Forward Protocol Specification
 // <https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1>`_.
 // [#extension: envoy.access_loggers.fluentd]
-// [#next-free-field: 7]
+// [#next-free-field: 8]
 message FluentdAccessLogConfig {
+  enum BackOffStrategy {
+    // Constructs fully jittered exponential backoff strategy.
+    // The backoff interval value will be in the range of [base_backoff_interval, max_backoff_interval).
+    JitteredExponential = 0;
+
+    // Constructs fully jittered backoff strategy based on the base_backoff_interval value.
+    // The backoff interval value will be in the range of [base_backoff_interval, 1.5 * base_backoff_interval).
+    JitteredLowerBound = 1;
+
+    // Constructs fixed backoff strategy based on the base_backoff_interval value.
+    Fixed = 2;
+  }
```

Comment:
Please use https://github.com/envoyproxy/envoy/blob/84dc857cea5ae0676403eac011dcf35ddaf9411d/api/envoy/config/core/v3/backoff.proto#L19 only. I don't think we need to add different strategies just for this? If we feel we do I would add those options to the common configuration so it is shared everywhere.

Reply:
Adding it to the common configuration would also require supporting in anywhere the backoff config is used, otherwise it's confusing?

Reply:
Correct. I don't really see why this case needs a special strategy but if you need it I would add it to the common place which shouldn't be very difficult as it is a common config. Feel free to do those changes in a separate PR to keep this one simpler.

Reply:
Done, I'll add the option to select the strategy type in a sepatare PR